llvm/test/tools/llvm-readobj/ELF/file-header-os-abi.test

```diff
@@ -167,7 +167,7 @@ FileHeader:
 # RUN: llvm-readelf --file-headers %t.osabi.amd.hsa | FileCheck %s --match-full-lines --check-prefix=OSABI-AMDHSA-GNU
 
 # OSABI-AMDHSA-LLVM: OS/ABI: AMDGPU_HSA (0x40)
-# OSABI-AMDHSA-GNU:  OS/ABI: 40
```

Comment:
Is there a bigger test diff?

Reply:
Nope, only one that changed. I could make some new ones but it doesn't seem like this is tested strenuously anywhere else so I figured it was considered fairly trivial.